charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     app.kubernetes.io/component: coordinator
 data:
   node.properties: |
-    node.environment={{ .Values.server.node.environment }}
+    node.environment={{ template "trino.environment" . }}
```

Comment:
Inline this template. Can you also update the tests to show how this could be used?